packages/react/src/shared/utils/withLoadPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import LoadPanel from 'devextreme-react/load-panel';
+
+interface WithLoadProps {
+    data: unknown;
```

Comment:
Let's try making it more generic: use the 'isLoading' prop instead of 'data' here. In this case, 'data' can have any name and type, and reusing will be easier.